en/test_and_ci/integration_testing_mavsdk.md

```diff
@@ -15,10 +15,10 @@ Install either from binaries or source:
 
 ## Prepare PX4 Code
 
-To build the PX4 code, use:
+To build PX4 source code for simulator testing, use:
 
 ```sh
-DONT_RUN=1 make px4_sitl gazebo mavsdk_tests
+DONT_RUN=1 make px4_sitl gazebo mavsdk_tests 
```

Comment:
Looks like it depends on gazebo. 
- Is this installed with the other pre-requisites listed?
- Doesn't this force a dependency on Linux?

Reply:
Yes it depends on Gazebo and Gazebo should be installed on Linux and macOS, I would think.

Reply:
Thanks. Modified the intro so that this is at least mentioned.